app/helpers/application_helper.rb

```diff
@@ -1,21 +1,28 @@
 module ApplicationHelper # :nodoc:
-  def markup(text, filters = [])
+  def markup(text, options={})
```

Comment:
Could we do something like

```suggestion
  def markup(text, *args)
```

Which would make the usage a bit cleaner. `markup(@issue.text, :liquid)` instead of `markup(@issue.text, liquid: true)`

Reply:
started there but abandoned, because the options can have nested values, in our case we're immediately using filters: [filter1, filter2]